client/src/pages/Lobby/Lobby.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, useEffect, useState } from 'react'
+import axios from 'axios'
+import { RouteComponentProps } from '@reach/router'
+import { Game } from '../../types/Game'
+import * as moment from 'moment'
+
+import './Lobby.scss'
+
+export interface LobbyProps extends RouteComponentProps {
+  id?: string
+}
+
+const Lobby: FC<LobbyProps> = (props: LobbyProps) => {
+  const [game, setGame] = useState<Game>()
+  useEffect(() => {
+    axios.get(`/games/${props.id}`).then(response => {
+      setGame(response.data as Game)
+    })
+  }, [props.id])
+
+  const getHostName = (game: Game) => {
+    const host = game.players.find(player => player.id === game.hostId)
+    return host ? host.name : 'host unknown'
+  }
+
+  const getSubtitle = (game: Game) => {
+    return `Créé par ${getHostName(game)} ${moment
+      .utc(game.creationDate)
+      .fromNow()}`
+  }
```

Comment:
why not directly declare a variable instead of functions to be called later ?

Reply:
Need to wait for api response. You would do this in the axios callback ? This seems fine to me 😄 .

Reply:
ah right !